3-enrich/stream-enrich/kinesis/src/main/scala/com.snowplowanalytics.snowplow.enrich.stream/sources/KinesisSource.scala

```diff
@@ -56,37 +58,43 @@ object KinesisSource {
       case c: Kinesis => c.success
       case _ => "Configured source/sink is not Kinesis".failure
     }
-    goodSink <-
-      KinesisSink.createAndInitialize(kinesisConfig, config.buffer, config.out.enriched, tracker)
-        .validation
-    threadLocalGoodSink = new ThreadLocal[Sink] {
-      override def initialValue = goodSink
-    }
-    badSink <-
-      KinesisSink.createAndInitialize(kinesisConfig, config.buffer, config.out.bad, tracker)
-        .validation
-    threadLocalBadSink = new ThreadLocal[Sink] {
-      override def initialValue = badSink
-    }
+    _ <- KinesisSink.validate(kinesisConfig, config.out.enriched).validation
```

Comment:
Don't we want to aggregate these? If `validate` fails on `config.out.enriched` we'll never know that `config.out.bad` is also invalid.